parsl/providers/kubernetes/kube.py

```diff
@@ -98,6 +102,8 @@ def __init__(self,
                  max_mem: str = "500Mi",
                  init_cpu: float = 1,
                  init_mem: str = "250Mi",
+                 extra_requests: Optional[Dict[str, str]] = {},
+                 extra_limits: Optional[Dict[str, str]] = {},
```

Comment:
I suggest adding requests and limits that you'd like to configure as init parameters, like we do with `max_cpu` and `init_mem`, rather than through an open-ended dict object.

Reply:
Ah, makes sense. Another option might be something like `hugepages` and `hugepages_size` in addition to a `custom_requests` and `custom_limits` parameters, but I'm cool with either route.